src/frontend/apps/drive/src/features/i18n/translations.json

```diff
@@ -174,6 +181,12 @@
           "unexpected": "Une erreur inattendue est survenue."
         }
       },
+      "roles": {
```

Comment:
Please place the key in the same order in french and english. For the english you put `roles` just above `explorer`, not in french